src/components/View/ViewHeader/CreateDocumentDialog.tsx

```diff
@@ -77,9 +79,9 @@ export const CreateDocumentDialog = ({ type, payload, children, mutator }: Props
 function getTemplate(type: View): (id: string) => Document {
   switch (type) {
     case 'Planning':
-      return templates.planning
+      return Templates.planning
     case 'Event':
-      return templates.event
+      return Templates.event
```

Comment:
What are the thoughts regarding the capital T? It's not a component, it's not a class?

Reply:
* **c**amelCase for all variables/consts and object members (properties, methods)
* **P**ascalCase for class names, constructors and static objects

Reply:
But this is not 100%. So I can discuss it... ;)

Reply:
I would say UPPERCASE for constants, but I don't really like that either....